rtrlib/rtr_mgr.c

```diff
@@ -58,21 +59,24 @@ static int rtr_mgr_start_sockets(struct rtr_mgr_group *group)
 	return RTR_SUCCESS;
 }
 
-int rtr_mgr_find_group(const struct rtr_mgr_config *config,
-		       const struct rtr_socket *sock,
-		       unsigned int *ind)
+static struct rtr_mgr_group *rtr_mgr_find_group(struct rtr_mgr_config *config,
+						const struct rtr_socket *sock)
```

Comment:
Weird, this looks fine in my editor and also on github when I view the whole file. But in this view there is indeed a space missing...